src/app/components/sites/pages/annotations/point.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, OnInit } from "@angular/core";
```

Comment:
this file probably should not be named `point` ... _point_ is a user term, not a technical term